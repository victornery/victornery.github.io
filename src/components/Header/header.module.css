.header {
  background-image: linear-gradient(to right, #24C6DC, #514A9D);
  color: var(--light)
}

.intro {
  position: absolute;
  top: 50%;
  left: 50%;

  width: 100%;
  max-width: 80%;
  font-size: 40px;
  
  text-align: center;
  transform: translate(-50%, -50%);
}