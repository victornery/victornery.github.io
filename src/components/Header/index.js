import React from 'react'
import styles from './header.module.css'

const Header = () => (
  <section className={styles.header}>
    <h1 className={styles.intro}>Victor Nery.</h1>
  </section>
)

export default Header