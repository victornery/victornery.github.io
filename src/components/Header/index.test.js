import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Header from './index'

const parent = document.createElement('div')

it('render without crashing', () => {
  ReactDOM.render(<Header />, parent)
  ReactDOM.unmountComponentAtNode(parent)
})

it('renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON()
  expect(tree).toMatchSnapshot()
});