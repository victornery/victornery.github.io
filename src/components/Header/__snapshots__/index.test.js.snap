// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="header"
>
  <h1
    className="intro"
  >
    Mussum Ipsum, cacilds vidis litro abertis. Admodum accumsan disputationi eu sit. Vide electram sadipscing et per. Casamentiss faiz malandris se pirulit√°.
  </h1>
</section>
`;
